<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Uitgooi</class>
 <widget class="QMainWindow" name="Uitgooi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Options</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="lblScheme">
               <property name="text">
                <string>Selsect Scheme:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cmbScheme"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="btnSelectSFolder">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select the RAAM folder of the school.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>Select Source Folder</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblSource">
             <property name="text">
              <string>Source:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnSelectDFolder">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select the file the final result will be placed in.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>Select Destination Folder</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblDestination">
             <property name="text">
              <string>Destination:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnSelectKFolder">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select the KLAS folder&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>Select Klas Foto Folder</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblKlasfotos">
             <property name="text">
              <string>Klas fotos:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Preview</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="imgPreview">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>161</width>
             <height>241</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btnBegin">
      <property name="text">
       <string>Begin</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progress">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
